environment.yml

```diff
@@ -11,5 +11,7 @@ dependencies:
   - eigen
   - spiceypy
   - numpy
+  - numba
```

Comment:
what is numba being used for?

Reply:
quaternion uses numba for acceleration. If you don't have it, it complains on import.

Reply:
I know this PR didn't add it, but eigen too I think.